src/language-js/types/estree.d.ts

```diff
@@ -0,0 +1,97 @@
+import * as ESTree from "estree";
+import * as Babel from "@babel/types";
+import { TSESTree } from "@typescript-eslint/types";
+import * as NGTree from "angular-estree-parser/lib/types";
+
+// WORKAROUND HACK for typescript-eslint issue:
+// https://github.com/typescript-eslint/typescript-eslint/issues/2388
+// MUST REMOVE to avoid errors with TypeScript 4.0.0:
+import * as ts from "typescript";
+declare module "typescript" {
+  type NamedTupleMember = Node;
```

Comment:
This will work even with **TypeScript 4.x**:
```suggestion
  export interface NamedTupleMember extends Node {}
```

Reply:
Thanks @ExE-Boss. I would actually favor keeping this workaround as-is, errors can remind us to remove it with the TypeScript 4 update. Nice idea, though.

Reply:
We should be able to remove this entire workaround if <https://github.com/typescript-eslint/typescript-eslint/pull/2405> is merged and released. I hope this will be possible.